clients/java/signalr/src/main/java/com/microsoft/signalr/HubConnection.java

```diff
@@ -55,18 +55,38 @@
     private final Logger logger = LoggerFactory.getLogger(HubConnection.class);
 
 
+    /**
+     * Sets the server timeout interval for the connection.
+     *
+     * @param serverTimeout The timeout duration.
+     */
     public void setServerTimeout(Duration serverTimeout) {
         this.serverTimeout = serverTimeout;
     }
 
+    /**
+     * Gets the server timeout duration.
+     *
+     * @return The server timeout duration.
+     */
     public Duration getServerTimeout() {
         return this.serverTimeout;
     }
 
+    /**
+     * Sets the connections Keep Alive Interval duration.
```

Comment:
Sets the ~~connections~~ ~~K~~**k**eep ~~A~~**a**live ~~I~~**i**nterval.

(avoid Title Case and we already know this relates to the connection; plus that would require an apostrophe if we did use it: `connection's`)